app/assets/stylesheets/projects.css.scss

```diff
@@ -66,3 +66,13 @@
   font-style: italic;
   padding-left: 20px;
 }
+
+.nocounter {
+    color: darkgray;
+}
+
+.project.nocounter, p.nocounter {
```

Comment:
@hugopl, these lines were added because I tried to unify two different places to put in gray those links that doesn't have MR's. I will remove then.